daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1104,7 +1106,9 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
 
 	// https://github.com/docker/docker/issues/4036
 	if supported := devicemapper.UdevSetSyncSupport(true); !supported {
-		logrus.Warn("Udev sync is not supported. This will lead to unexpected behavior, data loss and errors. For more information, see https://docs.docker.com/reference/commandline/cli/#daemon-storage-driver-option")
+		if !devices.overrideUdevSyncCheck {
+			logrus.Warn("Udev sync is not supported. This will lead to unexpected behavior, data loss and errors. For more information, see https://docs.docker.com/reference/commandline/cli/#daemon-storage-driver-option")
```

Comment:
good question ...  `devicemapper.UdevSetSyncSupport(true)` is needed, but for 1.7.1 we could can skip/comment-out `devices.overrideUdevSyncCheck` test here